api/jreleaser-resource-bundle/src/main/resources/org/jreleaser/bundle/Messages.properties

```diff
@@ -160,6 +160,7 @@ signing.file.newer                   = {} is newer than {}
 signing.check.published.key          = Checking if key {} has been published
 signing.key.published                = Key {} was found as published
 signing.key.not.published            = Key {} was NOT found as published
+signing.passphrase.blank             = Passphrase is blank. In case a passphrase is needed, you can configure a value using the Gradle DSL, or define a System property {}, or define a {} environment variable, or define a key/value pair in {} with a key named {}
```

Comment:
The Gradle DSL is not the only one. There should be another message key in the resources file that can be used as a guide.